doc/source/raster_class.md

```diff
@@ -360,8 +360,31 @@ rast_reproj.to_xarray()
 ```
 
 ## Obtain Statistics
-The `get_stats()` method allows to extract key statistical information from a raster in a dictionary.
-Supported statistics are : mean, median, max, mean, sum, sum of squares, 90th percentile, nmad, rmse, std.
+The {func}`~geoutils.Raster.get_stats` method allows to extract key statistical information from a raster in a dictionary.
+Supported statistics are :
+- **Mean:** arithmetic mean of the data, ignoring masked values.
+- **Median:** middle value when the valid data points are sorted in increasing order, ignoring masked values.
+- **Max:** maximum value among the data, ignoring masked values.
+- **Min:** minimum value among the data, ignoring masked values.
+- **Sum:** sum of all data, ignoring masked values.
+- **Sum of squares:** sum of the squares of all data, ignoring masked values.
+- **90th percentile:** point below which 90% of the data falls, ignoring masked values.
+- **LE90 (Linear Error with 90% confidence):** Difference between the 95th and 5th percentiles of a dataset, representing the range within which 90% of the data points lie. Ignore masked values.
+- **NMAD (Normalized Median Absolute Deviation):** robust measure of variability in the data, less sensitive to outliers compared to standard deviation. Ignore masked values.
+- **RMSE (Root Mean Square Error):** commonly used to express the magnitude of errors or variability and can give insight into the spread of the data. Only relevant when the raster represents a difference of two objects. Ignore masked values.
+- **Std (Standard deviation):** measures the spread or dispersion of the data around the mean, ignoring masked values.
+- **Valid points:** number of finite data points in the array. It counts the non-masked elements.
+- **Total points:** total size of the raster.
+- **Percentage valid points:** ratio between **Valid points** and **Total points**.
+
+
+If an inlier mask is passed:
+- **Inlier points:** number of unmasked data points in the inlier mask.
+- **Valid inlier points:** number of unmasked data points in the array after applying the inlier mask.
```

Comment:
Great!
Small final remarks:
- We could use "Total inlier points" instead of "Inlier points" to keep consistent naming,
- I think the description of "Inlier points" can be simplified to "Number of data points in the inlier mask" (no mention of mask as it includes both masked and unmasked).

And finally, my preference was towards the naming "Count" (used by NumPy, SciPy, etc) rather than "Points" which is ambiguous. I think @duboise-cnes had mentioned this too further above proposing "Number of points" instead, a longer version of "Count".

See you in a moment!